openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.upgrade.clicked.json

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:13bc41f6-8406-4006-b929-c9f1ac156ada",
+  "type": "NavigationEvent",
+  "actor": {
+    "type": "Person",
+    "id": "edx.devstack.lms:18000/u/honor"
+  },
+  "action": "NavigatedTo",
+  "object": {
+    "id": "http://edx.devstack.lms:18000/dashboard",
+    "name": "Open edX Enrollment Upgrade"
+  },
+  "eventTime": "2018-11-01T07:11:21.947626",
+  "extensions": {
+    "extra_fields": {
+      "user_id": 7,
+      "org_id": " ",
+      "edx_event_type": "edx.course.enrollment.upgrade.clicked",
```

Comment:
This key is simply `event_type` in the edx JSON. We do not need to change the key.